Networking/Networking/Requests/WordPressOrgRequest.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import Alamofire
+
+/// Represents a WordPress.org REST API Endpoint
+///
+struct WordPressOrgRequest: URLRequestConvertible {
```

Comment:
I think we need to update https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/NETWORKING.md#urlrequestconvertible about the new `URLRequestConvertible` implementation. 😄

Reply:
Also updated in 2d6aef5aac24954d9b29a5e5b1f6b66e38931023.